package_manifest.yaml

```diff
@@ -351,7 +351,7 @@ katello_packages:
         - "{{ nightly_releaser }}"
       build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
       nightly_package_tito_releaser_args:
-        - "--arg jenkins_job=katello-master-release"
+        - "--arg jenkins_job=rubygem-katello-nightly-release"
```

Comment:
This was the reason for the test failing oddly. We have not re-named this job yet.

Reply:
Looks like I was a bit too trigger happy.